examples/2016-10-31/image_binary_media_types/template.yaml

```diff
@@ -0,0 +1,68 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+
+Parameters:
+  ImageType:
+    Type: String
+    Default: image~1gif
+Globals:
+  Api:
+    # Send/receive binary data through the APIs
+    BinaryMediaTypes:
+      # These are equivalent to image/gif and image/png when deployed
+      - image~1jpg
+      - image~1gif
+      - image~1png
+      - application~1octet-stream
+
+Resources:
+  MyApi:
+    Type: AWS::Serverless::Api
+    Properties:
+      StageName: Prod
+      DefinitionBody:
+        swagger: "2.0"
+        info:
+          version: "2016-09-23T22:23:23Z"
+          title: "Simple Api"
+        basePath: "/none"
+        schemes:
+          - "https"
+        paths:
+          "/none":
+            get:
+              x-amazon-apigateway-integration:
+                httpMethod: POST
+                type: aws_proxy
+                contentHandling: CONVERT_TO_BINARY
+                passthroughBehavior: NEVER
+                uri:
+                  Fn::Sub: arn:${AWS::Partition}:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${MyLambda.Arn}/invocations
+              responses: {}
+
+  MyLambda:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: index.handler
+      Runtime: nodejs8.10
+      CodeUri:
+        Bucket: <YOUR_BUCKET>
```

Comment:
Could you make this work with SAM CLI? Have this point to the local code package then add instructions to use `sam package` and `sam deploy` to deploy this application.